onnxruntime/test/python/transformers/benchmark_mha.py

```diff
@@ -43,6 +57,20 @@ def get_name_list() -> List[str]:
         return ["Q,K,V", "QKV", "Q,KV", "Q,K',V'"]
 
 
+class SdpaKernel(IntEnum):
+    """Bit flags for sdpa_kernel CUDA provider option"""
+
+    DEFAULT = 0
+    FLASH_ATTENTION = 1
+    EFFICIENT_ATTENTION = 2
+    TRT_FUSED_ATTENTION = 4
+    CUDNN_FLASH_ATTENTION = 8
+    MATH = 16
```

Comment:
Should we call this `UNFUSED_ATTENTION` instead?

Reply:
I use similar naming as Torch SDPA, and existing c++ code:
https://github.com/microsoft/onnxruntime/blob/2862eb2db15293acaed626fbe6e64a67afdf6598/onnxruntime/contrib_ops/cpu/bert/attention_common.h#L156